mingus/midi/midi_file_in.py

```diff
@@ -215,11 +233,11 @@ def parse_midi_file_header(self, fp):
                 "Couldn't read number of tracks " "and/or time division from tracks."
             )
 
-        chunk_size -= 6
+        #chunk_size -= 6
         if chunk_size % 2 == 1:
             raise FormatError("Won't parse this.")
-        fp.read(chunk_size / 2)
-        self.bytes_read += chunk_size / 2
+#        fp.read(chunk_size // 2)
+#        self.bytes_read += chunk_size // 2
```

Comment:
Again, this seems like a debug thing. Indeed the divisions here should be integer divisions, but it seems to me there's no reason to remove this code and it should be uncommented.

Reply:
This might have been coming from LilyPond midi files - maybe worth keeping an eye on this in future